shared/src/business/entities/CaseDeadline.test.ts

```diff
@@ -28,54 +61,46 @@ describe('CaseDeadline', () => {
         },
         { applicationContext },
       );
-      expect(caseDeadline.getFormattedValidationErrors()).toEqual(null);
-    });
 
-    it('should be valid when all fields are present except for leadDocketNumber', () => {
-      const caseDeadline = new CaseDeadline(
-        {
-          associatedJudge: 'Judge Buch',
-          deadlineDate: '2019-03-01T21:42:29.073Z',
-          description: 'One small step',
-          docketNumber: DOCKET_NUMBER,
-        },
-        { applicationContext },
-      );
-      expect(caseDeadline.getFormattedValidationErrors()).toEqual(null);
+      expect(caseDeadline.getValidationErrors()).toEqual(null);
     });
 
-    it('should generate a sortableDocketNumber from the docketNumber passed in', () => {
+    it('should be valid when all required fields are present and optional fields are not', () => {
       const caseDeadline = new CaseDeadline(
         {
           associatedJudge: 'Judge Buch',
           deadlineDate: '2019-03-01T21:42:29.073Z',
           description: 'One small step',
           docketNumber: DOCKET_NUMBER,
+          leadDocketNumber: undefined, /// Optional property
         },
         { applicationContext },
       );
-      expect(caseDeadline.sortableDocketNumber).toEqual(2019000123);
+
+      expect(caseDeadline.getValidationErrors()).toEqual(null);
     });
 
-    it('should have error messages for invalid fields', () => {
+    it('should be invalid and return a helpful message when the user provides a description that is too long', () => {
       const caseDeadline = new CaseDeadline(
         {
           associatedJudge: 'Judge Buch',
           deadlineDate: '2019-03-01T21:42:29.073Z',
           description: `I got the horses in the back
-Horse tack is attached
-Hat is matte black
-Got the boots that's black to match
-Ridin' on a horse, ha
-You can whip your Porsche
-I been in the valley
-You ain't been up off that porch, now`,
+            Horse tack is attached
+            Hat is matte black
+            Got the boots that's black to match
+            Ridin' on a horse, ha
+            You can whip your Porsche
+            I been in the valley
+            You ain't been up off that porch, now`, // Description can be at most 120 characters
```

Comment:
Thank you for not changing this important part of our codebase. The lore remains intact.

Reply:
![nod](https://media.giphy.com/media/2jMjCYe570QdwsFMeI/giphy.gif)